Failed tests by Secp256k1

My env is the same as #58.

Error log:

```
$ cargo test --no-fail-fast
   Compiling aluvm v0.6.0 (/home/monaka/rgb/rust-aluvm)
error[E0599]: no variant or associated item named `Secp256k1` found for enum `Instr` in the current scope
  --> examples/asm.rs:25:16
   |
25 |       let code = aluasm! {
   |  ________________^
26 | |         clr     r1024[5]                        ;
27 | |         put     5,a16[8]                        ;
28 | |         putif   0xaf67937b5498dc,r256[1]        ;
...  |
92 | |         ret                                     ;
93 | |     };
   | |_____^ variant or associated item not found in `Instr<_>`
   |
   = note: this error originates in the macro `instr` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `aluvm` due to 4 previous errors
```

Is this issue related to #14 ?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,3 +33,7 @@ all = ["std", "secp256k1", "curve25519", "strict_encoding", "serde"]
 std = ["amplify/std", "bitcoin_hashes/std", "secp256k1/std", "curve25519-dalek/std", "curve25519-dalek/alloc", "bech32/std"]
 curve25519 = ["curve25519-dalek"]
 serde = ["serde_crate", "serde_with", "amplify/serde", "bitcoin_hashes/serde", "std", "secp256k1/std", "curve25519-dalek/serde"]
+
+[[example]]
+name = "asm"
+required-features = ["all"]

